.github/workflows/tacos_apply.yml

```
@@ -53,6 +53,8 @@ jobs:
 
       - id: slices
         uses: ./tacos-gha/.github/actions/determine-slices
+        with:
+          allowlist: apply.allowlist
```

This is a nice, neat solution. Good idea. It will simplify several aspects (including reader's clarity) measurably if we define this to be an explicit relative path though. How about:

```suggestion
        with:
          allowlist: .config/tacos-gha/allowlist/apply
```

I won't bikeshed you on the exact file naming though.